vue_frontend/src/App.vue

```diff
@@ -1,30 +1,7 @@
 <template>
-  <div id="nav">
-    <router-link to="/">Home</router-link> |
-    <router-link to="/about">About</router-link>
-  </div>
   <router-view/>
 </template>
 
 <style lang="scss">
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-}
-
-#nav {
-  padding: 30px;
-
-  a {
-    font-weight: bold;
-    color: #2c3e50;
-
-    &.router-link-exact-active {
-      color: #42b983;
-    }
-  }
-}
+  @import '../node_modules/bulma';
```

Comment:
```suggestion
  @import 'bulma';
```

Every dependency from `/node_modules` you can import with an "absolute path". [Just like you did with Axios](https://github.com/atosborges00/tako-CRM/pull/10/commits/4dc01b4eaabc62ac57aa878a002af3861503faec#diff-42bf1ccb838de89ed8436a7c757bceed6762fd285393ffefc060a00821dbb7e2R5):
```js
import axios from 'axios'
```